src/auth.rs

```diff
@@ -0,0 +1,141 @@
+use crate::openapi::apis::configuration::Configuration;
+use crate::openapi::apis::{authenticate_api, transactions_api};
+use crate::Error;
+
+pub struct Auth {
+    pub configuration: Configuration,
```

Comment:
Does this field need to be public? I’d expect it to be encapsulated in the struct after initialization, but let me know if I’m missing a use case.

Reply:
the parent `set_server_url` method is used in testing and needs to set each nested class's base path for the tests to mock correctly

https://github.com/passageidentity/passage-flex-rust/blob/b7160f9d857f65d0fbf587901f04b07c68ba74a5/src/passage_flex.rs#L58-L61

Reply:
i started messing with passing a reference, but it got kinda messy with the lifecycle annotations so decided to just make them pub